src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -75,6 +91,14 @@ public TupleTypeSymbol(ImmutableArray<Location> locations, NamedTypeSymbol under
             return Create(locationOpt, underlyingType, elementLocations, elementNames);
         }
 
+        public static TupleTypeSymbol Create(NamedTypeSymbol tupleCompatibleType)
+        {
+            return Create(ImmutableArray<Location>.Empty,
+                          tupleCompatibleType, 
+                          default(ImmutableArray<Location>), 
+                          default(ImmutableArray<string>));
+        }
```

Comment:
What location?

---
In reply to: [62076926](https://github.com/dotnet/roslyn/pull/11047#discussion_r62076926) [](ancestors = 62076926)